src/main/java/com/logsentinel/client/model/LogInfo.java

```diff
@@ -0,0 +1,31 @@
+package com.logsentinel.client.model;
+
+public class LogInfo {
```

Comment:
isn't this the same as MerkleTreeInfo? If yes, leave only one

Reply:
The wrong swagger API definition for the retrieval of basic log information caused confusion (wrong return type) and I did not found the appropriate class. But there it is. Going to fix this.